build-scripts/collect_remediations.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import os.path
+import argparse
+
+import ssg.build_remediations as remediation
+import ssg.build_yaml
+import ssg.rules
+import ssg.jinja
+import ssg.environment
+import ssg.utils
+import ssg.xml
+
+
+def parse_args():
+    p = argparse.ArgumentParser()
```

Comment:
What about describing the function that the script performs, i.e. that it puts all remediations, templated or not, to the output directory.